session/manager_http.go

```diff
@@ -49,12 +49,12 @@ func NewManagerHTTP(r managerHTTPDependencies) *ManagerHTTP {
 	}
 }
 
-func (s *ManagerHTTP) UpsertAndIssueCookie(ctx context.Context, w http.ResponseWriter, r *http.Request, ss *Session) error {
+func (s *ManagerHTTP) UpsertAndIssueCookie(ctx context.Context, w http.ResponseWriter, r *http.Request, ss *Session, cookieModifiers ...x.CookieModifier) error {
```

Comment:
Because the nonce is optional (passed as an argument), there are a couple of places where `UpsertAndIssueCookie` / `IssueCookie` are being called. There, the cookie would probably not be changed because the nonce is missing. Therefore, remove this parameter and just always set the nonce!

Reply:
Dropped parameter